### DISCUSSION TITLE
does rsbuild support vanilla js or ts project

### DISCUSSION THREAD

**Discussion by laoshaw** [web-infra-dev/rsbuild#3820] [Q&A] [answered]
that is, no framework, with linter and prettier etc ready to be used for vanilla js/html/css projects

**Comment by chenjiahan** [comment, marked as answer]
Yes, https://rsbuild.dev/guide/start/quick-start#creating-an-rsbuild-project

![image](https://github.com/user-attachments/assets/4ca91f9c-f899-4922-be0b-78deadb34473)
